support-frontend/cloud-formation/cfn.yaml

```
@@ -319,6 +317,24 @@ Resources:
     - TargetGroup
     - ElasticLoadBalancer
 
+  CatalogLoadingFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
+      AlarmName: Support Frontend Catalog Load Failure
+      MetricName: CatalogLoadingFailure
+      Namespace: support-frontend
+      Dimensions:
+        - Name: Environment
+          Value: PROD
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Threshold: 1
+      Period: 60
+      EvaluationPeriods: 1
+      Statistic: Average
+
```

```suggestion
TreatMissingData: notBreaching

```

I think we may need this because it [defaults to missing](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html#cfn-cloudwatch-alarms-treatmissingdata), which is probably not what we want (see: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html#alarms-and-missing-data)

---

Have discussed offline and we are going to leave this